connectors/D7SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,113 @@
+{
+  "basePath": "/secure",
+  "consumes": [],
+  "definitions": {},
+  "host": "rest-api.d7networks.com",
+  "info": {
+    "description": "D7 SMS allows you to reach your customers via SMS over D7's own connectivity to global mobile networks. D7 provides reliable and cost-effective SMS services to businesses across all industries and aims to connect all countries and territories via direct connections.",
+    "title": "D7SMS",
+    "version": "1.0"
+  },
+  "parameters": {},
+  "paths": {
+    "/balance": {
+      "get": {
+        "description": "Check account balance",
+        "operationId": "Balance",
+        "parameters": [],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "properties": {
+                "data": {
+                  "description": "data",
+                  "properties": {
+                    "balance": {
+                      "description": "balance",
+                      "type": "string"
+                    },
+                    "sms_count": {
+                      "description": "sms_count",
+                      "type": "string"
+                    }
+                  },
+                  "type": "object"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Check Balance"
+      }
+    },
+    "/secure/balance": {},
+    "/secure/send": {},
+    "/send": {
+      "post": {
+        "description": "Send SMS  to recipients using D7 SMS Gateway",
+        "operationId": "SendSMS",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "Body",
+            "required": false,
+            "schema": {
+              "properties": {
+                "content": {
+                  "description": "content",
+                  "type": "string"
+                },
+                "from": {
+                  "description": "from",
+                  "type": "string"
+                },
+                "to": {
+                  "default": 971562316353,
```

Comment:
Can you please remove this default value?

Reply:
Done